configure.in

```diff
@@ -313,6 +315,7 @@ AC_CHECK_LIB(rt, sem_init, , [
 ])
 AC_CHECK_LIB(posix4, sem_wait, , )
 AC_CHECK_LIB(pthread, pthread_create, , )
+AC_CHECK_LIB(flatccrt, flatcc_builder_init, , )
```

Comment:
This is just compilation.  I don't think it's unreasonable to force people to install a dependent library.